spyder/api/editorextension.py

```diff
@@ -1,14 +1,22 @@
 # -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013-2016 Colin Duquesnoy and others (see pyqode/AUTHORS.rst)
+# Copyright (c) 2016- Spyder Project Contributors (see AUTHORS.txt)
 #
-# Copyright © Spyder Project Contributors
-# Copyright © <2013-2016> <Colin Duquesnoy and others, see pyqode/AUTHORS.rst>
-# Licensed under the terms of the MIT License
-# (see spyder/__init__.py for details)
+# Distributed under the terms of the MIT License
+# (see NOTICE.txt in the Spyder root directory for details)
+# -----------------------------------------------------------------------------
 
 """
 This module contains the editor extension API.
-Adapted from https://github.com/pyQode/pyqode.core/blob/master/pyqode/core/api/mode.py
+
+Adapted from pyqode/core/api/mode.py of the
+`PyQode project <https://github.com/pyQode/pyQode>`_.
+Original file:
+<https://github.com/pyQode/pyqode.core/blob/master/pyqode/core/api/mode.py>
```

Comment:
Is this proper rst? I mean, does this line gets transformed to an actual link by Sphinx when shown in our Help plugin?

I say it because it seems you're using rst here but I'm so sure.